### ISSUE TITLE
Raspbian 1.17.3~2: FATA[0000] pulling image failed: rpc error: code = Unknown desc = Error choosing image instance: no image found in manifest list for architecture arm, variant v8, OS linux

### ISSUE THREAD

**Issue by pjoomen** [cri-o/cri-o#3626]
**Description**

After upgrading `cri-o-1.17` on Raspbian to `1.17.3~2` w/ custom aarch64/v8 kernel and standard armhf/32-bits userland the following error is received:

`FATA[0000] pulling image failed: rpc error: code = Unknown desc = Error choosing image instance: no image found in manifest list for architecture arm, variant v8, OS linux`

This is functional on the same system, using the cri-o binary from `1.17.2~2`.

**Steps to reproduce the issue:**
1. Run Raspbian Buster with a 64 bits kernel (configure `arm_64bit=1` in `/boot/config.txt` and reboot)
2. Install cri-o from the upstream Raspbian 10 repository
3. `crictl pull k8s.gcr.io/pause:3.1`

**Describe the results you received:**

```
# crictl pull k8s.gcr.io/pause:3.1
FATA[0000] pulling image failed: rpc error: code = Unknown desc = Error choosing image instance: no image found in manifest list for architecture arm, variant v8, OS linux
```

**Describe the results you expected:**

```
# crictl pull k8s.gcr.io/pause:3.1
Image is up to date for k8s.gcr.io/pause@sha256:c84b0a3a07b628bc4d62e5047d0f8dff80f7c00979e1e28a821a033ecda8fe53
```

**Output of `crio --version`:**

```
crio version 1.17.3
commit: "3473adcfe09a306a2ac15773fc7c3f06dc5ff9db"
```

**Comment by haircommander** [comment]
I think this would be fixed by using `k8s.gcr.io/pause:3.2`

**Comment by pjoomen** [comment]
No, it does not, but it was the first thing i tried.

**Comment by haircommander** [comment]
PTAL @vrothberg @nalind

**Comment by vrothberg** [comment]
@mtrmac is working on a fix afaik.

**Comment by saschagrunert** [comment]
I guess the fix is about how to relax the matching between variant and architecture, right?

**Comment by vrothberg** [comment]
Yes

**Comment by mtrmac** [comment]
For the record, this is tracked in https://github.com/containers/image/issues/898 .

**Comment by haircommander** [comment]
this should be fixed in 1.17 and 1.18

**Comment by haircommander** [comment]
closing, please reopen if you disagree
